src/components/forms/PlanForm/helpers.ts

```diff
@@ -719,16 +747,10 @@ export function getPlanFormValues(planObject: PlanDefinition): PlanFormFields {
     }
   }
 
-  let taskGenerationStatus: taskGenerationStatusType;
-
-  if (isDynamicPlan(planObject)) {
-    taskGenerationStatus = taskGenerationStatuses[2]; // Disabled
-  } else {
-    taskGenerationStatus =
-      taskGenerationStatusContext.length > 0
-        ? (taskGenerationStatusContext[0].valueCodableConcept as taskGenerationStatusType)
-        : taskGenerationStatuses[1];
-  }
+  const taskGenerationStatus: taskGenerationStatusType =
```

Comment:
probably needs a check to give dynamicPlans a status of disabled if one is not given